pgx-utils/src/sql_entity_graph/pg_extern/attribute.rs

```diff
@@ -129,6 +111,21 @@ impl Parse for Attribute {
                 let _bracket = syn::bracketed!(content in input);
                 Self::Requires(content.parse_terminated(PositioningRef::parse)?)
             }
+            "sql" => {
+                use syn::Lit;
+
+                let _eq: Token![=] = input.parse()?;
+                match input.parse::<syn::Lit>()? {
```

Comment:
I imagine this could also handle a `syn::Path`.

Reply:
When the path was expected as a string, that wasn't the case, but with your other suggested change, this will be one of the affected parses.